src/types.ts

```diff
@@ -140,8 +140,8 @@ export interface RawTechnicalQuestion {
   question_id: string;
   title: string;
   created: string;
-  created_by: Member;
-  approved_by: Member;
+  created_by: string;
+  approved_by: string;
```

Comment:
same here

Reply:
Yes but I'm returning it as a string of the actual name with the serializer.